<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mess Management System</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="logoLabel">
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="text">
       <string>LOGO HERE</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="signOutButton">
      <property name="text">
       <string>Sign Out</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <!-- Mess In/Out Tab -->
      <widget class="QWidget" name="messTab">
       <attribute name="title">
        <string>Mess In/Out</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="messStatusGroup">
          <property name="title">
           <string>Mess Status</string>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="statusLabel">
             <property name="text">
              <string>Current Status:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="currentStatusLabel">
             <property name="text">
              <string>Not Messed In</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="lastMessInLabel">
             <property name="text">
              <string>Last Mess In:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="lastMessInTimeLabel">
             <property name="text">
              <string>Never</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="messInButton">
            <property name="text">
             <string>Mess In</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="messOutButton">
            <property name="text">
             <string>Mess Out</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <!-- Menu Tab -->
      <widget class="QWidget" name="menuTab">
       <attribute name="title">
        <string>Menu</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTableWidget" name="menuTable"/>
        </item>
       </layout>
      </widget>
      <!-- Track Record Tab -->
      <widget class="QWidget" name="trackTab">
       <attribute name="title">
        <string>Track Record</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QGroupBox" name="duesGroup">
          <property name="title">
           <string>Current Dues</string>
          </property>
          <layout class="QFormLayout" name="formLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Total Dues:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="totalDuesLabel">
             <property name="text">
              <string>0.00</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="messHistoryTable"/>
        </item>
        <item>
         <widget class="QPushButton" name="printBillButton">
          <property name="text">
           <string>Print Bill</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <!-- Menu Suggestions Tab -->
      <widget class="QWidget" name="suggestionsTab">
       <attribute name="title">
        <string>Menu Suggestions</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QGroupBox" name="suggestionGroup">
          <property name="title">
           <string>Add Suggestion</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QTextEdit" name="suggestionInput"/>
           </item>
           <item>
            <widget class="QPushButton" name="submitSuggestionButton">
             <property name="text">
              <string>Submit Suggestion</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="suggestionsTable"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui> 